templates/analysis.yml

```diff
@@ -49,22 +26,29 @@ parameters:
 - name: angular       # <-- Not successfully tested; see note above
   type: boolean       #     (default is a Maven build when angular = false
   default: false      #      the Angular build steps will substitute when angular = true)
+- name: sonarKeyPrefix
+  type: string
+- name: sonarNamePrefix
+  type: string
+- name: sonarOrg
+  type: string
+- name: sonarUrlPrefix
+  type: string
+- name: discordChannel
+  type: string
+- name: discordKey
+  type: string
```

Comment:
fyi json is a subset of yaml and it's possible to make these fit on one line in an arguably more-readable way if you prefer, using what yaml calls "flow style". e.g.:
```
- {name: discordChannel, type: string}
- {name: discordKey, type: string}
```
a valid yaml parser is supposed to interpret that as identical to the more-common "block style" syntax, so anything reading the file shouldn't be able to tell the difference.

Reply:
I prefer to keep it YAML, but that's good to know!